indico/worker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.9
+FROM python:3.9-bullseye
```

Comment:
We could use `python:3.10-bullseye`, no?

Reply:
We could use 3.11, and replace centos7 with something less ancient for postgres. But as discussed on mattermost this is probably something for the "big" PR we plan to merge when releasing the 3.3 image in the future